src/fragments/lib/geo/js/geofences.mdx

```diff
@@ -52,3 +52,328 @@ initializeMap();
 ![A Geofences Management Application](/images/geofence-management-app.png)
 
 ## Geofence API
+
+If you are using a different mapping library or have a need for a more programmatic approach to managing geofences, the `@aws-amplify/geo` package provides methods for managing geofences, but not geofence collections.
+
+First, you need to import Geo from either the `@aws-amplify/geo` or `aws-amplify` packages.
+
+```javascript
+import { Geo } from "@aws-amplify/geo";
+// Or
+import { Geo } from "aws-amplify";
+```
+
+### saveGeofences
+
+`saveGeofences` is used to save geofences to your collection. It can take a single geofence or an array of geofences.
+
+#### API
+
+```javascript
+Geo.saveGeofences(geofences, options);
+```
+
+#### Parameters
+
+- `geofences` - can be a single geofence object, or an array of geofence objects to save to a collection.
+
+Geofence objects must have the following properties:
+
+- `geofenceId` - a opaque and unique identifier for the geofence.
+- `geometry` - a geometry object that defines the geofence.
+  - `polygon` - an array of arrays with [Longitude, Latitude] coordinates.
+
+<Callout>
+
+NOTE: Polygon arrays have a few requirements:
+
+- must have at least 4 vertices (i.e. 4 coordinate points)
+- the first and last point must be the same in order to complete the polygonal loop
+- vertices must be in counter-clockwise order
+
+</Callout>
+
+```json
+{
+  "geofenceId": "string",
+  "geometry": {
+    "polygon": [
+      [
+        [longitude, latitude],
+        [longitude, latitude],
+        [longitude, latitude],
+        [longitude, latitude]
+      ]
+    ]
+  }
+}
+```
+
+- `options` - optional options object for saving geofences
+  - `collectionName` - the name of the collection to save geofences to.
+    - Defaults to the default collection listed in your `aws-exports.js` file after provisioning a geofence collection resource.
+
+```js
+{
+  collectionName: "myCollection";
+}
+```
+
+#### Return
+
+The return from `saveGeofences` is a Promise that resolves to an object with both successes and errors for geofences that were successfully created or not.
+
+Each success object has the following properties:
+
+- `geofenceId` - the geofenceId of the geofence that was saved.
+- `createTime` - the time the geofence was created.
+- `updateTime` - the time the geofence was last updated.
+
+Each error object has the following properties:
+
+- `geofenceId` - the geofenceId of the geofence that failed to be saved.
+- `error` - an error object
+  - `code` - the error code
```

Comment:
Need to add the potential values for developers to code against. https://docs.aws.amazon.com/location-geofences/latest/APIReference/API_BatchItemError.html ??